sdk/src/Services/DynamoDBv2/Custom/DataModel/Utils.cs

```diff
@@ -55,13 +57,12 @@ internal static class Utils
         };
 
         public static readonly IEnumerable<Type> PrimitiveTypes = new HashSet<Type>(primitiveTypesArray);
-        private static readonly HashSet<ITypeInfo> PrimitiveTypeInfos = new HashSet<ITypeInfo>(primitiveTypesArray
-            .Select(p => TypeFactory.GetTypeInfo(p)));
+        private static readonly HashSet<Type> PrimitiveTypeInfos = new HashSet<Type>(primitiveTypesArray
```

Comment:
I think you can rename `PrimitiveTypeInfos`.

Reply:
I actually just removed `PrimitiveTypeInfos`. It was only used once doing `Any` operation that the other `PrimitiveTypes` field works just fine for.